cfn-resources/cluster/cmd/resource/resource.go

```diff
@@ -100,23 +90,24 @@ func Create(req handler.Request, _ *Model, currentModel *Model) (handler.Progres
 	}
 
 	// Create Cluster
-	cluster, res, err := client.AdvancedClusters.Create(context.Background(), *currentModel.ProjectId, clusterRequest)
+	cluster, res, err := client.AtlasV2.ClustersApi.CreateCluster(context.Background(), *currentModel.ProjectId, clusterRequest).Execute()
 	if err != nil {
 		_, _ = log.Warnf("Create - Cluster.Create() - error: %+v", err)
```

Comment:
I would reduce the number of logs in this PR, for example in this case the error info is already in the response

Reply:
updated